src/batch/src/executor/hash_agg.rs

```diff
@@ -207,9 +207,9 @@ impl<K: HashKey + Send + Sync> HashAggExecutor<K> {
                 });
 
                 // TODO: currently not a vectorized implementation
-                states
-                    .iter_mut()
-                    .for_each(|state| state.update_single(&chunk, row_id).unwrap());
```

Comment:
Oops. I believe there're more cases of unwrapping the result in closure in our system, just for no idea of how to throw the error. 😢

Reply:
Yes.😢